src/main/java/org/jvnet/hudson/test/InboundAgentRule.java

```diff
@@ -455,7 +455,7 @@ public String run(JenkinsRule r) throws Throwable {
             return options.getName();
         }
 
-        @SuppressFBWarnings(value = {"PATH_TRAVERSAL_IN", "NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE"}, justification = "just for test code")
```

Comment:
`NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE` was a lie at any rate - if `computer` was null we would never return!